.gitignore

```diff
@@ -26,12 +26,17 @@ project/plugins/project/
 .idea.bak/
 intellij/out
 
+# Vim
```

Comment:
can you please remove these changes, it's unrelated to your functionality.

Reply:
Though these are not related to functionality, I'd still rather leave them, as they are not invasive minor improvements that may be useful for other future contributors of the project who happen to use Vim, Metals or Jenv.

Reply:
of course. please add them in a separate pull request.

Reply:
#2248